CMakeLists.txt

```diff
@@ -162,8 +162,13 @@ if(ASPECT_WITH_WORLD_BUILDER)
     set(WORLD_BUILDER_SOURCE_DIR "${CMAKE_SOURCE_DIR}/contrib/world_builder/" CACHE PATH "" FORCE)
   endif()
 
-  # add source and include dirs:
+  # Add the WorldBuilder include dirs. This includes both the WorldBuilder
+  # header files, as well as where the cmake run we do below puts the
+  # WorldBuilder config.h file.
   include_directories("${WORLD_BUILDER_SOURCE_DIR}/include/")
+  include_directories("${CMAKE_BINARY_DIR}/world_builder/include")
```

Comment:
If we run in debugrelease mode this will use the debug version of this header for both targets. I am wondering if we can do something like
```
target_include_directories(aspect "${CMAKE_BINARY_DIR}/world_builder/include")
target_include_directories(aspect-release "${CMAKE_BINARY_DIR}/world_builder_release/include")
```